content/assets/scripts/halloween/worker.js

```diff
@@ -0,0 +1,123 @@
+var face_cascade, eye_cascade;
+
+function loadFaceDetectTrainingSet() {
+	if (face_cascade == undefined) {
+		face_cascade = new cv.CascadeClassifier();
+		let load = face_cascade.load('../../test/data/haarcascade_frontalface_default.xml');
```

Comment:
Shouldn't this file be committed as well?

Reply:
That html file was obsolete, just like some code in worker.js, I removed everything that was not required with the latest commit.